docs/source/models/gbd2017/causes/cause_model_template.rst

```diff
@@ -91,125 +82,193 @@ Disease Description
-    something else?
 
-    In `Pull Request 91`_, @James said:
+Cause Hierarchy
++++++++++++++++
 
-      I don't think it needs the cause name. Something like `GBD Modeling
-      Strategy` seems fine.
+Restrictions
+++++++++++++
 
-    Also for this section, @James said:
+The following table describes any restrictions in GBD 2017 on the effects of
+this cause (such as being only fatal or only nonfatal), as well as restrictions
+on the ages and sexes to which the cause applies.
+
+.. list-table:: GBD 2017 Cause Restrictions
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     -
+     -
+   * - Female only
+     -
+     -
+   * - YLL only
+     -
+     -
+   * - YLD only
+     -
+     -
+   * - YLL age group start
+     -
+     -
+   * - YLL age group end
+     -
+     -
+   * - YLD age group start
+     -
+     -
+   * - YLD age group end
+     -
+     -
+
+
+Vivarium Modeling Strategy
+--------------------------
 
-      Point people to yld and cod appendices for their diseases and link to
-      http://ghdx.healthdata.org/gbd-2017/code which can serve as an annoying
-      to read but authoritative source on how the cause was actually modeled.
+Scope
++++++
 
+.. todo::
 
-Cause Hierarchy
-+++++++++++++++
+  Describe which aspects of the disease this cause model is designed to
+  simulate, and which aspects it is **not** designed to simulate.
+
+Assumptions and Limitations
++++++++++++++++++++++++++++
+
+.. todo::
+
+  Describe the clinical and mathematical assumptions made for this cause model,
+  and the limitations these assumptions impose on the applicability of the
+  model.
+
+.. admonition:: Todo for template development
+
+  What else goes in the `Vivarium Modeling Strategy` section? Should there be
+  any other subsections besides `Scope` and `Assumptions and Limitations`?
 
 Cause Model Diagram
 -------------------
 
-State and Transition Data Tables
---------------------------------
+Data Description
+----------------
 
 .. admonition:: Todo for template development
 
-    - Decide on section heading names and structure. E.g. should this section be
-      called "Data Description" instead of "State and Transition Data Tables,"
-      like it was originally?
+  Decide on section heading name(s) and structure for the data section. See
+  comments by @Beatrix and @Nathaniel in `Pull Request 64`_ discussing the
+  following questions:
 
-      In `Pull Request 91`_, @James said:
+  - Should this section be called "Data Description" as it is now, or would
+    something like "State and Transition Data Tables" be more descriptive?
 
-        I think `Data Description` is fine.
+    In `Pull Request 91`_, @James said:
+
+      I think `Data Description` is fine.
 
-    - Is there anything else that belongs in a "Data Description" section
-      besides the data tables?
+  - Should this section have subsections (such as "State and Transition Data
```

Comment:
I like state and transition data tables, but either is fine in my opinion

Reply:
you can give the tables titles.  Don't need the extra subsections

Reply:
I meant, should there be more than one subsection, one of which would be the tables. If the tables are the only thing in the "data description" section, then I agree with Ali (and Beatrix in PR 64) that it would be more descriptive to change the name of the whole section to "State and Transition Data Tables" since that clarifies what data we're talking about.